services/command-router-quarkus/src/main/java/org/eclipse/hono/commandrouter/quarkus/Application.java

```diff
@@ -288,10 +293,10 @@ private CommandRouterService commandRouterService() {
             final CommandTargetMapper commandTargetMapper) {
 
         final MessagingClientProvider<CommandConsumerFactory> commandConsumerFactoryProvider = new MessagingClientProvider<>();
-        if (kafkaProducerConfig.isConfigured() && kafkaConsumerConfig.isConfigured()) {
+        if (commandInternalKafkaProducerConfig.isConfigured() && kafkaConsumerConfig.isConfigured()) {
```

Comment:
Shouldn't this be `commandResponseKafkaProducerConfig` instead of `commandInternalKafkaProducerConfig`? The latter one is checked again a couple of lines below...
In general, I think the checks for Kafka configuration could be improved but that is independent of this PR.

Reply:
I've changed the condition now to also include `commandResponseKafkaProducerConfig`.